tools/disasm/disasm.py

```diff
@@ -263,7 +440,11 @@ def str_round(string, precision):
         return int_part + "." + frac_part
 
     def to_binary(flt_str):
-        return as_dword(struct.pack(">d", float(flt_str))) if is_double else as_word(struct.pack(">f", float(flt_str)))
+        return (
+            as_dword(struct.pack(">d", float(flt_str)))
+            if is_double
+            else as_word(struct.pack(">f", float(flt_str)))
+        )
```

Comment:
I think redability would be benefited if one liners like this wouldn't be used in functions like this one

```suggestion
        if is_double:
            return as_dword(struct.pack(">d", float(flt_str)))
        return as_word(struct.pack(">f", float(flt_str)))
```